.github/workflows/rust.yml

```diff
@@ -0,0 +1,21 @@
+name: Rust
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose --no-default-features # disable sdl
+    - name: Run tests
+      run: cargo test --verbose --no-default-features # disable sdl
```

Comment:
Do you have to disable SDL? I thought we could link with it, we just didn't get a graphics context? Right?